tests/test_prompt.py

```diff
@@ -167,6 +209,38 @@ def test_create_prompt_with_null_config():
     assert prompt.config == {}
 
 
+def test_get_prompt_by_version_or_label():
+    langfuse = Langfuse()
+    prompt_name = create_uuid()
+
+    for i in range(3):
+        langfuse.create_prompt(
+            name=prompt_name,
+            prompt="test prompt " + str(i + 1),
+            labels=["production"] if i == 1 else [],
+        )
+
+    default_prompt_client = langfuse.get_prompt(prompt_name)
+    assert default_prompt_client.version == 2
+    assert default_prompt_client.prompt == "test prompt 2"
+    assert default_prompt_client.labels == ["production"]
+
+    first_prompt_client = langfuse.get_prompt(prompt_name, 1)
+    assert first_prompt_client.version == 1
+    assert first_prompt_client.prompt == "test prompt 1"
+    assert first_prompt_client.labels == []
+
+    second_prompt_client = langfuse.get_prompt(prompt_name, version=2)
```

Comment:
![image](https://github.com/langfuse/langfuse-python/assets/17686849/57d6246c-44c3-4315-99c2-6adcd1784901)
Do you have the same signature when hovering?

Reply:
do we need `version` as `arg` for backwards compatibility while all others are `kwargs`?